index.html

```diff
@@ -313,14 +313,19 @@ <h2>Managing Focus and Supporting Keyboard Navigation</h2>
-          <p>When implementing <pref>aria-activedescendant</pref> as described below, the user agent keeps the <abbr title="Document Object Model">DOM</abbr> focus on the container element but communicates <a>desktop focus events</a> and states to the assistive technology as if the active descendant has focus. User agents are not expected to validate that the active descendant is a descendant of the focused container. It is the responsibility of the user agent to ensure that keyboard events are processed at the container <a>element</a> that has <abbr title="Document Object Model">DOM</abbr> focus. Any keyboard events directed at the active descendant bubble up to the <abbr title="Document Object Model">DOM</abbr> container element with focus for processing.</p>
+          <p>
+            When implementing <pref>aria-activedescendant</pref> as described below, the user agent keeps the <abbr title="Document Object Model">DOM</abbr> focus on the container element or on an input element that controls the container element.
+            However, the user agent communicates <a>desktop focus events</a> and states to the assistive technology as if the element referenced by <pref>aria-activedescendant</pref> has focus.
+            User agents are not expected to validate that the active descendant is a descendant of the container element.
+            It is the responsibility of the user agent to ensure that keyboard events are processed at the <a>element</a> that has <abbr title="Document Object Model">DOM</abbr> focus.
+            Any keyboard events directed at the active descendant bubble up to the <abbr title="Document Object Model">DOM</abbr> element with focus for processing.
+          </p>
 	  <section id="managingfocus_authors">
 	    <h3>Information for Authors</h3>
 	    <p>If the author removes the element with focus, the author SHOULD move focus to a logical element. Similarly, authors SHOULD not scroll the element with focus off screen unless the user performed a scrolling action.</p>
 	    <p>Authors SHOULD ensure that all interactive <a>elements</a> are focusable and that all parts of composite widgets are either focusable or have a documented alternative method to achieve their function.</p>
 	    <p>Authors MUST manage focus on the following container roles:</p>
 	    <ul>
```

Comment:
> Authors MUST manage focus on the following container roles:

Doesn't combobox need to stay in this list? Authors do still need to manage focus on them.
Perhaps change the list intro sentence to read:

> Authors MUST manage focus on the following roles:

And add `<li><rref>combobox</rref></li>` back into the list?

Reply:
No, this is one of the more important changes. It changes combobox from a composite, which manages focus, to an input. Inputs include button, radio, textbox, etc. These elements do not have focusable descendants.

authors manage focus in the popup that is controlled by the combobox, but there is nothing to manage in the combobox itself. In this way, combobox becomes much like a menubutton. It also makes it more like a native select.

Reply:
> authors manage focus in the popup that is controlled by the combobox, but there is nothing to manage in the combobox itself.

Ok. I was thinking that "managing `aria-activedescendant` in the combobox" was the same thing as "managing focus in the combobox".